src/attribute_set.rs

```diff
@@ -242,6 +242,7 @@ macro_rules! evdev_enum {
                 serializer.serialize_str(value)
             }
         }
+        #[cfg(feature = "serde")]
         paste::paste! {
             #[cfg(feature = "serde")]
```

Comment:
you don't need this second cfg on line 247 anymore since the first should remove the entire paste section?